anchor/common/ssv_types/src/committee.rs

```diff
@@ -0,0 +1,33 @@
+use crate::OperatorId;
+use alloy::primitives::keccak256;
+use derive_more::{Deref, From};
+
+const COMMITTEE_ID_LEN: usize = 32;
+
+/// Unique identifier for a committee
+#[derive(Clone, Copy, Debug, Default, Eq, PartialEq, Hash, From, Deref)]
+pub struct CommitteeId(pub [u8; COMMITTEE_ID_LEN]);
+
+impl From<Vec<OperatorId>> for CommitteeId {
```

Comment:
If we'd like to simplify the caller code we could implement sth like:

```rust
impl From<&IndexSet<OperatorId>> for CommitteeId {
    fn from(cluster_members: &IndexSet<OperatorId>) -> Self {
        let mut sorted: Vec<_> = cluster_members.iter().copied().collect();
        sorted.sort();
        let mut data: Vec<u8> = Vec::with_capacity(sorted.len() * 4);
        for id in sorted {
            data.extend_from_slice(&id.to_le_bytes());
        }
        keccak256(data).0.into()
    }
}
```

Then call` let committee_id: CommitteeId = (&self.cluster_members).into();`